.github/workflows/validate.yaml

```diff
@@ -60,7 +66,10 @@ jobs:
 
       - name: Regenerate all generated files
         working-directory: ./src/github.com/${{ github.repository }}
-        run: make generated-files
+        env:
+          REGISTRY_REDHAT_IO_USERNAME: ${{ secrets.REGISTRY_REDHAT_IO_USERNAME }}
+          REGISTRY_REDHAT_IO_PASSWORD: ${{ secrets.REGISTRY_REDHAT_IO_PASSWORD }}
+        run: make generated-files generate-catalog
```

Comment:
I'd include `generate-catalog` in `generated-files` so that others don't need to learn about this new command to run ?

Reply:
Pushed the split.

Reply:
wow 1/2 hour

Reply:
Alright. In github action it's much faster. About 5 minutes:
```
11:28:54.752 INFO:    Generating catalog for OCP 4.13
time="2024-10-11T11:28:54Z" level=info msg="rendering index \"registry.redhat.io/redhat/redhat-operator-index:v4.13\" as file-based catalog"
time="2024-10-11T11:29:49Z" level=info msg="wrote rendered file-based catalog to \"/tmp/knative.aIjkdvzz/tmp.IiMQGlnAOi\"\n"
11:30:27.698 INFO:    Generating catalog for OCP 4.14
time="2024-10-11T11:30:27Z" level=info msg="rendering index \"registry.redhat.io/redhat/redhat-operator-index:v4.14\" as file-based catalog"
time="2024-10-11T11:31:21Z" level=info msg="wrote rendered file-based catalog to \"/tmp/knative.aIjkdvzz/tmp.s4bv1h7HZi\"\n"
11:31:39.478 INFO:    Generating catalog for OCP 4.15
time="2024-10-11T11:31:39Z" level=info msg="rendering index \"registry.redhat.io/redhat/redhat-operator-index:v4.15\" as file-based catalog"
time="2024-10-11T11:32:31Z" level=info msg="wrote rendered file-based catalog to \"/tmp/knative.aIjkdvzz/tmp.vOKEBnWFRH\"\n"
11:32:48.077 INFO:    Generating catalog for OCP 4.16
time="2024-10-11T11:32:48Z" level=info msg="rendering index \"registry.redhat.io/redhat/redhat-operator-index:v4.16\" as file-based catalog"
time="2024-10-11T11:33:36Z" level=info msg="wrote rendered file-based catalog to \"/tmp/knative.aIjkdvzz/tmp.mKSK5ppXVI\"\n"
11:33:57.680 INFO:    Generating catalog for OCP 4.17
time="2024-10-11T11:33:57Z" level=info msg="rendering index \"registry.redhat.io/redhat/redhat-operator-index:v4.17\" as file-based catalog"
time="2024-10-11T11:34:12Z" level=info msg="wrote rendered file-based catalog to \"/tmp/knative.aIjkdvzz/tmp.TB6GDPKkiP\"\n"
```